### ISSUE TITLE
Page Content: Why Decanter

### ISSUE THREAD

**Issue by buttonwillowsix** [SU-SWS/decanter#572]
https://elegant-poitras-87214a.netlify.com/page/about-why-decanter/

I think we should roll up the three pages below into one page:
-Accessibility
-Brand Compliance
-Get Started Quickly

**Comment by buttonwillowsix** [comment]
Passing to @kerri-augenstein for editing and review (and maybe also @sherakama and David)

**Comment by kerri-augenstein** [comment]
@sherakama over to you now for a look-over, edits, and additions.

**Comment by sherakama** [comment]
We have a fair bit of content on accessibility, ie, Nav research by ucomm, nav research by SWS, and other findings and documentation. Perhaps that section should be pulled out as put as stand alone.

**Comment by kerri-augenstein** [comment]
Ahhh yes, agreed @sherakama. Totally forgot we have a full page devoted to Accessibility and we'll need another one on Usability as well. Tweaked the outline paragraph on the "Why Decanter" page.

**Comment by sherakama** [comment]
Done.
